runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -825,6 +849,17 @@ private CorfuRuntime(@Nonnull CorfuRuntimeParameters parameters) {
             MetricsUtils.metricsReportingSetup(
                     defaultMetrics, parameters.getPrometheusMetricsPort());
         }
+        CorfuRuntimeParameters.MicroMeterRuntimeConfig microMeterRuntimeConfig =
+                this.parameters.getMicroMeterRuntimeConfig();
+        if (microMeterRuntimeConfig.metricsEnabled) {
+            LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
+            registry = Optional.ofNullable(loggerContext.exists(microMeterRuntimeConfig.configuredLoggerName))
+                    .map(logger -> MeterRegistryProvider.MeterRegistryInitializer.newInstance(logger,
```

Comment:
If `metricsEnabled = true` but no logger was found then we should print a warning message.

Reply:
done